### ISSUE TITLE
The plug-in failed and an error occurred after the player entered the server.

### ISSUE THREAD

**Issue by User-Time** [TheDutchMC/SkinFixer#63]
**Describe the bug**

**To Reproduce**
An error will appear after the player joins the server.
（I don’t know if this has anything to do with me using a VPN?）

**Expected behavior**
No error is reported, the player's skin is replaced

**Screenshots**
![image](https://github.com/TobiasDeBruijn/SkinFixer/assets/55058887/e08b84f7-1d8a-4f56-a016-d0d49edebae2)

**Version information (please complete the following information):**
 - Minecraft version: 1.20.4
 - SkinFixer version: 1.7.9
 - Operating system and OS version: Windows11
 - Architecture: 64

**Additional context**
My "clash for windows" is in TUN mode

**Comment by TobiasDeBruijn** [comment]
Right. I think this issue stems from the API having some serious availability problems. I'll need to see if I can figure out a way for users to self-host the API, though I am very limited on time at the moment.
If you want to take a crack at it yourself, the API can be found [here](https://github.com/TobiasDeBruijn/SkinFixer-API).
